snafu/log_generator_wrapper/trigger_log_generator.py

```diff
@@ -22,9 +22,12 @@
 import json
 import subprocess
 import uuid
+import os
 from kafka import KafkaConsumer, TopicPartition
 
 logger = logging.getLogger("snafu")
+if (os.getenv("snafu_disable_logs", "False")) == "True":
```

Comment:
Can we make this a parameter to the wrapper instead of a Env var?

Reply:
It shouldn't matter which way it is consumed right? Is there a reason for this preference? I would much rather have it as an env var because I can use it within the pod's command/args to quiesce any bash echos as well.

Reply:
Predominately it is to keep consistent with the other options that are in use in the wrapper but if it is going to hinder your workflow its fine.

Reply:
> Predominately it is to keep consistent with the other options that are in use in the wrapper but if it is going to hinder your workflow its fine.

Great, thanks!

Reply:
I don't see many users doing this at all and we also have a sane default. So perhaps not adding to the args at all, was my take. Also, to be able to use it inside the bash script in the pod.